project/LinkerdBuild.scala

```diff
@@ -585,11 +585,12 @@ object LinkerdBuild extends Base {
       .dependsOn(Protocol.thrift % "test", Interpreter.perHost % "test")
 
     val main = projectDir("linkerd/main")
-      .dependsOn(admin, configCore, core)
+      .dependsOn(admin, configCore, core, Protocol.http % "e2e", Interpreter.namerd % "e2e", Interpreter.perHost % "e2e")
       .withTwitterLib(Deps.twitterServer)
```

Comment:
What is the reason for adding `Protocol.http % "e2e", Interpreter.namerd % "e2e", Interpreter.perHost % "e2e"`? Where this tests that weren't added to the End to End test suite?

Reply:
I added an end-to-end test in linkerd/main which uses the http protocol, the namer interpreter, and the per-host interpreter.  Therefore I need to depend on those projects in the e2e configuration.